src/routes/GroupSettings/InviteSettingsTab/InviteSettingsTab.js

```diff
@@ -127,89 +127,113 @@ ${props.group.name} is using Hylo for our online group: this is our dedicated sp
       resendInvitation(invitationToken)
     }
 
-    return <div>
+    return <div styleName='styles.container'>
       <div styleName='styles.header'>
         <div styleName='styles.title'>Invite People</div>
-        <div styleName='styles.subtitle'>
-          to {name} on Hylo
-        </div>
       </div>
+
       {pending && <Loading />}
-      {!pending && <div styleName='styles.invite-link-settings'>
-        <div styleName='styles.invite-link-text'>
-          <div styleName='styles.help'>Anyone with this link can join the group</div>
-          {inviteLink && <div styleName='styles.invite-link'>{inviteLink}</div>}
-          {!inviteLink && <div styleName='styles.help'>No link has been set yet</div>}
-        </div>
-        <div styleName='styles.buttons'>
-          <Button onClick={onReset}
-            color={buttonColor(reset)}
-            styleName='styles.reset-button'
-            narrow
-            small>
-            {reset
-              ? 'Reset'
-              : (inviteLink ? 'Reset Link' : 'New Link')}
-          </Button>
-          {inviteLink && <CopyToClipboard text={inviteLink} onCopy={onCopy}>
-            <Button color={buttonColor(copied)} styleName='styles.copy-button' narrow small>
-              {copied ? 'Copied' : 'Copy Link'}
+
+      {!pending && <>
```

Comment:
what is this <>?

Reply:
https://reactjs.org/docs/fragments.html#short-syntax